release_dashboard/lib/logic/cherrypicks.dart

```diff
@@ -0,0 +1,9 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// Function that converts the cherrypick captured by input forms from string to
+// a string array
+List<String> cherrypickStringtoArray(String? cherrypickString) {
+  return cherrypickString == '' || cherrypickString == null ? <String>[] : cherrypickString.split(',');
```

Comment:
should we validate this input with our regex?

Reply:
Yes, it is being validated in this [PR](https://github.com/flutter/cocoon/pull/1423).

Reply:
I mean in this function

Reply:
maybe at least leave a TODO here and then implement an additional validation here when you land https://github.com/flutter/cocoon/pull/1423 (assuming you land this one first and rebase)

Reply:
There is one [here](https://github.com/Yugue/cocoon/blob/80846d7d8c51aea9eb6207acda0fd2efebccef26/release_dashboard/lib/widgets/create_release_substeps.dart#L99). The validations in #1423 will make sure the strings are valid before hitting this `cherrypickStringtoArray` function